language/guidelines/user-experience/performance/index.md

```diff
@@ -0,0 +1,73 @@
+
+Short-term memory is commonly cited to only be able to hold [7 ± 2](http://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two) chunks of information at a time, and that if more information is entered in to the short-term memory in that time, retrieval of information stored earlier becomes error prone. Short term memory has also been found to decay and only last around 18 seconds unless an individual is actively trying to retain a piece of information.
+
+Working memory's capacity has been hard to pin down exactly, but the two most prominent models, when taken together, suggest that the more information that must be maintained in working memory, the slower and more error prone concurrent process become and that the rate of decay of working memory is based on the temporal density and attentional demands of the processing task (this is also known as *cognitive load*).
+
+### The Size and Speed of Immersion
+
+Sensory memory, short-term memory, and working memory combine to determine how immersive an interface feels to a user. The faster a interface is able to respond to a user, the faster that user's sensory memory can pick up the response, the faster that response can be stored in short-term memory, and the faster working memory can start processing the result in context of the input. The longer that initial step takes, or the larger the amount being asked for and retrieved, the more likely that a user's memory for the task at hand will decay. This decay is what causes *web stress* and leads users abandon tasks and sites entirely.
+
+## Notifying Users About Interface Delays
+
+With an understanding of memory in hand, along with Jakob's research, users should be notified about duration of tasks in the following ways:
+
+* **<=100ms** - If a response takes 100ms or less, no notification is needed
+* **>100ms** - If a response takes more than 100ms, an indication that a user's response is being processed should be provided. This is commonly a loading indicator or (ideally) a progress bar showing percent complete
+* **>3s** - If a response takes more than 3s, the task should be backgrounded, giving the user the ability to subscribe to updates for when the task completes. Backgrounded tasks should include context of the original request in order to assist the user recall the reason the request was made.
+
+## Creating Smooth Interfaces
+
+A human's sensory memory, acting at 100ms, is able to identify around 10 unique images per second. In order to create the illusion of smooth movement for users, as opposed to a series of independent images, more images are needed. The number of images per second, or frame rate, is often expressed as either Frames per Second (FPS), or in hertz (Hz) when working with progressive scan monitors. For a frame of reference, the *de facto* standard for movies and television with sound is 24 FPS.
```

Comment:
Let's try to avoid using "frame" in this sentence, just to minimize confusion.

"For a **point** of reference, the de facto..."